js/components/ladderPage/ladder.tsx

```diff
@@ -185,6 +185,10 @@ const themeForVehicleOnLadder = (
   vehicle: Vehicle,
   ladderConfig: LadderConfig,
 ): TrainTheme => {
+  if (!vehicle.vehiclePosition.revenue) {
```

Comment:
suggestion: can we `=== false` this, since `revenue: null` should probably not display this?

Reply:
we discussed in real life- sounds good!